src/app/start/start.page.ts

```diff
@@ -25,14 +25,28 @@ export class StartPage implements OnInit {
 
   user = this.authService.getUser();
 
+  // Vr world
+  // to only allow admins to see pages related to VR games
+  userRole: String;
+
+
   constructor(
     public navCtrl: NavController,
     public toastController: ToastController,
     private _translate: TranslateService,
     private authService: AuthService
-  ) {}
+  ) { }
 
   async ngOnInit() {
+
+    // Get user role
+    this.user.subscribe(
+      event => {
+        if (event != null) {
+          this.userRole = (event['roles'])[0];
```

Comment:
you should store all user roles in an `userRoles` array and later on check wether it includes the corresponding role. we can of course also create a `vr` user role

Reply:
I will update it later, as for know I have only one 'admin' user. 
For now you can put 'admin' as first role in the array.